docs/validators/node-configuration.md

```diff
@@ -0,0 +1,84 @@
+- `config.toml`  holds the main configuration of the node (storers & cachers type and size, type of hasher, type of marshaller and so on)                                                  
+- `economics.toml` holds the economics configuration (such as genesis total supply, inflation per year, developer fees and so on)                                                            
+- `enableEpochs.toml` holds a list of new features or bugfixes and the epoch when they become active                                                                                            
+- `enableRounds.toml` holds a list of new features or bugfixes and the round when they become active                                                                                            
+- `external.toml` holds external drivers' configuration (for example: Elasticsearch or event notifier)                                                                                       
+- `gasSchedules` is the directory that contains all the gas schedules to be used for SC execution, depending on activation epochs specified on external.toml->GasSchedule->GasScheduleByEpochs 
+- `genesisContracts` is the directory that contains the WASM contracts that were deployed at Genesis time                                                                                                
+- `genesis.json`  contains all the addresses and their balance/active delegation at Genesis time                                                                                             
+- `genesisSmartContracts.json` specifies the SCs to be deployed at Genesis time, alongside additional parameters                                                                                           
+- `nodesSetup.json` holds all the Genesis nodes public keys, alongside their wallet address                                                                                                    
+- `p2p.toml` holds peer-to-peer configurable values, such as the number of peers to connect to                                                                                          
+- `prefs.toml` holds a set of custom configuration values, that should not be replaced from an upgrade to another                                                                         
+- `ratings.toml` holds the parameters used for the nodes' rating mechanism, for example the start rating, decrease steps, and so on                                                         
+- `systemSmartContractsConfig.toml` holds configuration values for System Smart Contracts, such as the Staking, ESDT, or Governance                                                                            
+
+### Overloading config.toml values 
+
+As mentioned in the above descriptions, `prefs.toml` should not be overwritten when performing an upgrade. 
```

Comment:
`prefs.toml is not overwritten by the installation scripts when ...`

Reply:
fixed